### DISCUSSION TITLE
Activate filter by button?

### DISCUSSION THREAD

**Discussion by AndMe2** [rappasoft/laravel-livewire-tables#937] [Q&A]
Hi!
Is there a way to activate all the the filters att once by implementing a button to click?
(Would like to start with an empty table, then setting the filters and apply all the filters with a button (and then load the model that the filters would apply on), to minimize the load.)

**Comment by mrqaidi** [comment]
i need this too

**Reply by lrljoe** [reply]
When you say "apply all the filters with a button", what do you mean.

Let's say you have two "Text Filter" filters, and a "Multi Select Dropdown Filter" filter.

What do you want to happen?

There are methods already to set a filter's value, both within the Component, and using Events.

**Reply by hoan-anh** [reply]
Old thread but I think what they meant is a button to simultaneously trigger all the filters.

It probably works like a normal, non-Livewire form component: you change the value of the filters > page not update > you hit the button > page gets updated, all the filters apply their new values.
